moment/moment-node.d.ts

```diff
@@ -465,7 +465,7 @@ declare module moment {
         max(moments: Moment[]): Moment;
 
         normalizeUnits(unit: string): string;
-        relativeTimeThreshold(threshold: string, limit: number): void;
+        relativeTimeThreshold(threshold: string, limit?: number): void|number;
```

Comment:
Sure, I'm pretty new to typescript, but should have had a better look at the rest of the file. 
Updated.